server/manager/src/main/java/org/apache/accumulo/manager/compaction/coordinator/CompactionCoordinator.java

```diff
@@ -409,7 +409,9 @@ private boolean canReserveCompaction(TabletMetadata tablet, CompactionJob job,
 
     switch (job.getKind()) {
       case SYSTEM:
-        if (tablet.getSelectedFiles() != null
+        if (tablet.getCompactionRequested()) {
```

Comment:
If you have not done so already, you will need to trace this back to where the TabletMetadata object is created to ensure that it contains your new column

Reply:
> If you have not done so already, you will need to trace this back to where the TabletMetadata object is created to ensure that it contains your new column

I thought I had done this but I missed that I needed to add the column to be fetched in TabletManagementIterator which was uncovered in my latest test update as it failed until I fixed this.